var/spack/repos/builtin/packages/googletest/package.py

```diff
@@ -49,12 +55,13 @@ def cmake_args(self):
         args = [
             self.define_from_variant("BUILD_SHARED_LIBS", "shared"),
             self.define_from_variant("CMAKE_CXX_STANDARD", "cxxstd"),
+            self.define_from_variant("BUILD_GMOCK", "gmock"),
+            self.define_from_variant("GTEST_HAS_ABSL", "absl"),
+            self.define("gtest_disable_pthreads", spec.satisfies("~pthreads")),
         ]
-        args.append(self.define("gtest_disable_pthreads", not spec.satisfies("+pthreads")))
-        if spec.satisfies("@1.8:"):
-            # New style (contains both Google Mock and Google Test)
+
+        if spec.satisfies("@:1.8.0"):
             args.append(self.define("BUILD_GTEST", True))
```

Comment:
This line is now applied for inverted logic (also fencepost warning: `@1.8:` is not the inverse of `@:1.8.0`).

Reply:
Right, this is a fix for an unused variable: the `BUILD_GTEST` flag was only required up to and including 1.8.0, and the flag was removed in 1.8.1. (I dug through the git history to verify this.)